app/controllers/v0/disability_compensation_in_progress_forms_controller.rb

```diff
@@ -28,10 +28,8 @@ def data_and_metadata_with_updated_rated_disabilites
           metadata['returnUrl'] = '/disabilities/rated-disabilities'
         end
         evss_rated_disabilites = JSON.parse(rated_disabilities_evss.rated_disabilities.to_json)
-        prased_form_data['ratedDisabilities'] = camelize_with_olivebranch(evss_rated_disabilites)
-        metadata['ratedDisabilitiesUpdated'] = true
+        prased_form_data['updatedRatedDisabilities'] = camelize_with_olivebranch(evss_rated_disabilites)
```

Comment:
I see that prased_form_data is a typo but it appears to be the same copy/paste error elsewhere

Reply:
🤦 Thanks!